src/clone.rs

```diff
@@ -360,16 +356,22 @@ macro_rules! clone {
             }
         }
     );
+    ($($(@ $strength:ident)? $($variables:ident).+ $(as $rename:ident)?),+ => @default-panic, move || $body:expr ) => (
+        clone!($($(@ $strength)? $($variables).+ $(as $rename)?),* => @default-panic, move || { $body })
+    );
     ($($(@ $strength:ident)? $($variables:ident).+ $(as $rename:ident)?),+ => $(@default-return $return_value:expr,)? move || $body:block ) => (
         {
             $( $crate::to_type_before!($(@ $strength)? $($variables).+ $(as $rename)?); )*
             move || {
-                let return_value = || $crate::to_return_value!($($return_value)?);
-                $( $crate::to_type_after!($(as $rename)? $(@ $strength)? $($variables).+, return_value );)*
+                let _return_value = || $crate::to_return_value!($($return_value)?);
```

Comment:
Why the underscore?

Reply:
In case you don't use the returned value, it avoids getting a warning about it (discovered it when adding the tests).

Reply:
Ah, makes sense. Merge when the CI is green please, or I'll merge it later once I notice that the CI has passed :)

I'll also do a backport for 0.9.1 then and then we can release, or what do you think?

Reply:
Sounds good to me!